packages/react-server/src/ReactFizzHooks.js

```diff
@@ -834,6 +834,7 @@ export const HooksDispatcher: Dispatcher = supportsClientAPIs
       useActionState,
       useFormState: useActionState,
       useHostTransitionStatus,
+      useMemoCache,
```

Comment:
Should `useCacheRefresh` be added here too or not?

Reply:
See https://github.com/facebook/react/pull/32068